Newer cmake has CHECK_INCLUDE_FILES() instead of check_include_files()
and needs to INCLUDE(CheckIncludeFiles) to use that function.

--- CMakeLists.txt	2013-08-04 06:22:20.000000000 +0200
+++ CMakeLists.txt	2016-09-19 12:56:04.548697684 +0200
@@ -35,11 +35,12 @@
 set(KU_FIRSTGID 500 CACHE STRING "First GID of normal users")
 set(KU_USERPRIVATEGROUP true CACHE STRING "User private groups by default")
 
-check_include_files(shadow.h HAVE_SHADOW_H)
-check_include_files(crypt.h HAVE_CRYPT_H)
-check_include_files(sys/stat.h HAVE_SYS_STAT_H)
-check_include_files(unistd.h HAVE_UNISTD_H)
-check_include_files(paths.h HAVE_PATHS_H)
+INCLUDE(CheckIncludeFiles)
+CHECK_INCLUDE_FILES(shadow.h HAVE_SHADOW_H)
+CHECK_INCLUDE_FILES(crypt.h HAVE_CRYPT_H)
+CHECK_INCLUDE_FILES(sys/stat.h HAVE_SYS_STAT_H)
+CHECK_INCLUDE_FILES(unistd.h HAVE_UNISTD_H)
+CHECK_INCLUDE_FILES(paths.h HAVE_PATHS_H)
 check_library_exists(crypt crypt "" HAVE_CRYPT_LIBRARY)
 
 set(CMAKE_REQUIRED_INCLUDES pwd.h)
